Update streaming section with parallel promises (#3521)
withastro/docs: src/content/docs/en/guides/server-side-rendering.mdx

</html>
```

In this example, `A name` will render while `personPromise` and `factPromise` are loading.

> Once `personPromise` has resolved `A fact` will be appear and `factPromise` will render when its finished loading.

Reviewer: I did a small wording change and fixed a typo, let me know what you think!
```suggestion
Once `personPromise` has resolved, `A fact` will appear and `factPromise` will render when it's finished loading.
```

Author: Love it, I've added a commit the suggestions. Thanks for the feedback.
